lib/private/Collaboration/Reference/LinkReferenceProvider.php

```diff
@@ -116,7 +116,9 @@ private function fetchReference(Reference $reference): void {
 			return;
 		}
 		$linkContentType = $headResponse->getHeader('Content-Type');
-		if ($linkContentType !== 'text/html') {
+		$expectedContentType = 'text/html';
+		// check the header begins with the expected content type
+		if (substr($linkContentType, 0, strlen($expectedContentType)) !== $expectedContentType) {
```

Comment:
with php 8.0 and higher we could use:

```suggestion
		if (!str_starts_with($linkContentType, $expectedContentType)) {
```

Reply:
Is 26 dropping Php 7.4 support?

Reply:
I thought so? Not?

Reply:
Maybe it's not up-to-date info but https://docs.nextcloud.com/server/latest/admin_manual/installation/system_requirements.html doesn't say so :grin:.

Reply:
wasnt there a mail from andy regarding this recently?

Reply:
> Maybe it's not up-to-date info but https://docs.nextcloud.com/server/latest/admin_manual/installation/system_requirements.html doesn't say so 😁.

yeah, is not up-to-date IIRC...

Reply:
https://github.com/nextcloud/server/blob/master/lib/versioncheck.php#L28

Let's wait for 27 to really put stuff that requires Php >= 8.0 :grin:

Reply:
cc @come-nc

Reply:
Also this should be backportable ;)

Reply:
true, fine by me then